packages/platform/src/lib/form-group.ts

```diff
@@ -195,6 +196,15 @@ export function createFormGroup<FormFields extends FormGroupCreator>(
       }
       Object.values(fg).forEach((f) => markFormControlAsTouched(f));
     },
+    markAsPristine: () => {
```

Comment:
To keep things consistent I would like to rename this to `markAllAsPristine` (similar to `markAllAsTouched`).